src/hotspot/share/opto/node.hpp

```diff
@@ -1128,6 +1128,23 @@ class Node {
   // Set control or add control as precedence edge
   void ensure_control_or_add_prec(Node* c);
 
+  // Visit boundary uses of the node and apply a callback function for each.
+  // Recursively traverse uses, stopping and applying the callback when
+  // reaching a boundary node, defined by is_boundary.
+  // Note: the function definition appears after the complete type definition
+  // of Unique_Node_List.
```

Comment:
Nit: generally we try to make the comments in "block" style. You could also make the lines a bit wider.

Reply:
Are you referring to the line break between the first sentence and the `Note:`? I'm aligning on 80 columns, but will of course change to the agreed upon convention.